Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/CheckInMatching.kt

```diff
@@ -0,0 +1,75 @@
+package de.rki.coronawarnapp.presencetracing.risk
```

Comment:
What about moving these global functions into a class so they can be mocked better when writing unit tests for `CheckInWarningMatcher`?

Reply:
I actually do not want them to be mocked in the CheckInWarningMatcherTest to have a test that covers the whole unit of work

Reply:
Should we make them part of that class? Why are they global functions in an extra file?

Reply:
I love Kotlin for offering global functions 😃 I can easily write unit tests for all kinds of edge cases without having to mock anything. And I can call them easily from everywhere, e.g the test menu. I can of course move them into the same file.

Reply:
I agree that they are very useful, but we run the risk of littering the Android Studio auto completion with functions that are only relevant in specific circumstances.

Scoping functions so only those that are interesting are available at specific places in the code make it easier for new developers to work on something, find relevant functions.

It's may also have a negative impact on IDE performance if there many global functions, similar to overuse of unscoped extension functions.

It would not be a blocker for me if you feel strongly about this.

Reply:
I marked them a internal, only visible for testing (for now)